demisto_sdk/commands/common/hook_validations/integration.py

```diff
@@ -156,6 +161,16 @@ def is_valid_as_deprecated(self):
         ]
         return all(answers)
 
+    def is_unskipped_integration(self):
+        """Validated the integration testing is not skipped."""
+        skipped_integrations = self._load_conf_file().get('skipped_integrations', {})
```

Comment:
loading `conf.json` for each item can be expensive, should be loaded once in the validate_manager.